module/criteria/helper.js

```diff
@@ -459,4 +468,80 @@ module.exports = class criteriaHelper {
     })
 }
 
+
+  /**
+   * Create copy of criterias
+   * @method
+   * @name createCopyOfCriterias
+   * @param {Array} criteriaIds - Array of criteria Id's        
+   * @returns {Object}  old and new Mapped criteria ids .  
+  */
+
+  static createCopyOfCriterias(criteriaIds= []) {
```

Comment:
@deepakr123 - Please rename it to **duplicate**

Reply:
resolved